packages/testlab/package.json

```diff
@@ -4,7 +4,8 @@
   "description": "A collection of test utilities we use to write LoopBack tests.",
   "main": "index",
   "scripts": {
-    "build": "tsc",
+    "build": "tsc && npm run build:es2015",
```

Comment:
Note that the current `tsconfig.common.json` has target `es6` (which is an alias for es2015). It makes it easy to run `ts-node` on Node 6, therefore I think it's better to keep it for now.

I think we should have `build:es2015` and `build:es2017`, and then `npm run build` can invoke both of them.